packages/mdc-linear-progress/README.md

```diff
@@ -101,3 +101,10 @@ MDC Linear Progress exposes the following methods:
 | `set reverse(value: boolean) => void` | Reverses the direction of the linear progress indicator. |
 | `open() => void` | Puts the component in the open state. |
 | `close() => void` | Puts the component in the closed state. |
+
+### Sass Mixins
```

Comment:
I'm thinking we should move this to just under the CSS Modifiers section since it's CSS-related

Reply:
Done!